Plugins/Public/conn_plugin/Main.cpp

```diff
@@ -178,24 +176,27 @@ void MoveClient(unsigned int client, unsigned int targetBase)
 		return;
 
 	// No plugin handled it, do it ourselves.
-	unsigned int system;
-	pub::Player::GetSystem(client, system);
 	Universe::IBase* base = Universe::get_base(targetBase);
-
-	pub::Player::ForceLand(client, targetBase); // beam
-
-	// if not in the same system, emulate F1 charload
```

Comment:
what happened to this, why do we no longer have this?

Reply:
I wondered what purpose this elaborate code that was emulating a F1 relog was, but when I got rid of it, I haven't noticed any changes, so I decided to get rid of 'excess fat'

Reply:
I've done a spot of archaeology. It's old code taken from the depths of HkBeam.
It appeared some time since v1.5.5 in 2007, but pre-dates the TSP FLHook repository from March 2010.
If the mtime on src/flhook_src/source/HkFuncPlayers.cpp in https://forge.the-starport.net/attachments/download/10/flhook1.6.0_plugin.rar is to be believed it dates from 03/10/08 or earlier.
I've done some more digging and found https://the-starport.net/freelancer/forum/viewtopic.php?post_id=4425#forumpost4425 which might be the best explanation we're going to get
Having read that I'd prefer not to remove it

Reply:
Agreed